docs/cloud/features/observability/development_environment.md

```diff
@@ -0,0 +1,73 @@
+
+**Simplified Communication and Team Alignment**:
+
+The development environment streamlines team communication through shareable URLs, eliminating the need for manual methods like PRs or DMs. These URLs serve as comprehensive summaries, displaying crucial information including last run times, data intervals for incremental models, and detailed change information such as metadata modifications and model removals.
+
+## Using the page
+The Environments page shows an overview of all the environments that exist in your project (both yours and any your teammates have created).
+
+![tcloud environment page](./development_environment/environments.png)
+
+The page's table includes a link to each environment's page, along with the environment's creation date, the date it was last updated, and the date it will expire if not updated again.
+
+### Selecting and Individual Environment page
+
+Clicking an environment's name from the main environments page takes you to its individual page.
+
+The page begins with an at-a-glance summary of the most recent plan applied to the environment, including:
+
+- Its completion status and time
+- The latest time interval backfilled by the plan
+- Count of models present in the environment
+- An interactive visualization that summarizes the differences between the environment's models and the `prod` environment's models
+    - The count of directly modified models is represented in blue
+    - The count of added models is green
+    - The count of removed models is red
+
+![tcloud development environment](./development_environment/tcloud_development_environment.png)
+
+??? "ProTip:"
+
+    If a stakeholder or else anyone on your team is looking to understand an environment you own and are working on, you can share the URl link with them and they will be abel to access and see all of the information about your environment. 
+    
+    Its a great place to start to have open conversations about what was recently added, removed or changed in a environment! 
+
+
+## Differences from Prod section
+
+Development environments are used to prepare and test changes before merging them to `prod`.
+
+The `Differences From prod` section provides a summary of model differences between the environment and `prod`, with separate tabs for directly and indirectly modified models.
+
+In the summary, each model's name is a link to [its model page](./model.md).
+
+## Plan history information
+
+The plan applications chart is a calendar visualization of all plans that have been applied to the environment in the previous 2 weeks.
+
+The chart represents time across days on its `x-axis`, where each column represents one day. The date corresponding to each day is displayed at the top of the chart.
+
+The chart represents time within a day on its `y-axis`, where each day begins at the top and ends at the bottom.
```

Comment:
Can we combine the `x-axis` and `y-axis` into one paragraph and remove the duplicated phrasing?

Reply:
addressed in [04708d1](https://github.com/TobikoData/sqlmesh/pull/3515/commits/04708d1b43fdfdff8f6b28aeb5590ee613ba0c7b)